llvm-remote-index.md

```diff
@@ -1,13 +1,11 @@
-# Unofficial LLVM remote index service
+# LLVM remote index service
 
-A clangd [remote index](/remote-index.html) server for the LLVM project is
-currently available for testing.
-The service is unsupported and is not an official LLVM product.
+A clangd [remote index](/remote-index.html) server for the [LLVM
+project](https://github.com/llvm/llvm-project) is currently available.
 
 To use it, you'll need clangd with remote-index support. Set the server address
-to `llvm.clangd-index-staging.dev:50051`, and the project root to your
-`llvm-project` directory. Details on how to do this are at
-[remote index](/remote-index.html).
+to `clangd-index.llvm.org:5900`, and the project root to your `llvm-project`
```

Comment:
i wonder if we should include an example config file directly in here, instead of adding another layer of indirection. e.g:

Put the following under ~/.config/clangd/config.yaml
```YAML
If:
  PathMatch: /path/to/llvm/.*
Index:
  External:
    Server: clangd-index.llvm.org:5900
    MountPoint: /path/to/llvm/
```